includes/Core/Modules/Modules.php

```diff
@@ -271,15 +267,6 @@ function( $token_response ) {
 						}
 					}
 				}
-
-				try {
-					$analytics = $this->get_module( Analytics_4::MODULE_SLUG );
-					// @TODO: We don't have this method in Analytics_4 module.
-					// Investigate if we need to add this method in Analytics_4 module or not.
-					// $analytics->handle_token_response_data( $token_response );
-				} catch ( Exception $e ) {
```

Comment:
Verified, this is only UA related
<img width="745" alt="Screenshot 2024-02-19 at 11 42 00" src="https://github.com/google/site-kit-wp/assets/10414808/b00e75c1-12d4-4a59-9362-87ba6e389989">